api/server.js

```diff
@@ -1,4 +1,5 @@
-/* eslint-disable no-underscore-dangle */
+/* eslint-disable no-console */
+/* eslint no-unused-vars: ["error", { "argsIgnorePattern": "^_" }] */
```

Comment:
why? what are these for?

Reply:
They are for eslint. I have turned on automatic eslint check for all pull requests. Now eslint has errors set for console use and for unused arguments. This rule disables alerts for console and disables unused alert for any argument that begins with an underscore.

Reply:
I know the linter disables, I just couldn't see what they correspond to in the code. oh well.